stable/clamav/Chart.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: v1
-appVersion: "1.6"
+apiVersion: v1.0.1
```

Comment:
There is no `apiVersion` of `v1.0.1`. This should be `v1`. Did you mean to changes something else?